The test_ncache_* functions require a working user directory, which is
generally unavailable in xbps-src builds.

--- a/src/tests/cmocka/test_negcache.c
+++ b/src/tests/cmocka/test_negcache.c
@@ -1371,6 +1371,7 @@
         cmocka_unit_test_setup_teardown(test_sss_ncache_domain_locate_type,
                                         setup, teardown),
 
+#if 0
         /* user */
         cmocka_unit_test_setup_teardown(test_ncache_nocache_user,
                                         test_ncache_setup,
@@ -1423,6 +1424,7 @@
         cmocka_unit_test_setup_teardown(test_ncache_both_gid,
                                         test_ncache_setup,
                                         test_ncache_teardown),
+#endif
     };
 
     tests_set_cwd();
